applications/StructuralMechanicsApplication/custom_utilities/tangent_operator_calculator_utility.h

```diff
@@ -103,35 +109,39 @@ class TangentOperatorCalculatorUtility
         )
     {
         // Converged values to be storaged
-        const Vector& strain_vector_gp = rValues.GetStrainVector();
-        const Vector& stress_vector_gp = rValues.GetStressVector();
+        const Vector strain_vector_gp = rValues.GetStrainVector();
```

Comment:
I would use the constructor if possible

and name this variable as something like "saved_strain" or "unperturbed_strain" (same with the stress)